src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -1079,6 +1079,34 @@ public final class JavadocTokenTypes {
 
     /**
      * Equals html tag component: {@code '='}.
+     *
+     * <p><b>Example:</b></p>
+     *
+     * <pre>{@code
+     * &lt;div class="container" id="main"&gt;&lt;/div&gt;
+     * }</pre>
+     *
+     * <p><b>Tree:</b></p>
+     * <pre>{@code
+     * HTML_TAG -> HTML_TAG
+     *  |--START -> <
+     *  |--HTML_TAG_NAME -> div
+     *  |--WS ->
+     *  |--HTML_TAG_NAME -> class
+     *  |--EQUALS -> =
+     *  |--ATTR_VALUE -> "container"
+     *  |--WS ->
+     *  |--HTML_TAG_NAME -> id
+     *  |--EQUALS -> =
+     *  |--ATTR_VALUE -> "main"
+     *  `--END -> >
+     * }</pre>
+     *
+     * @see
+     * <a href="https://docs.oracle.com/javase/8/docs/technotes/tools/unix/javadoc.html">
+     * Oracle Docs</a>
```

Comment:
link should be https://docs.oracle.com/javase/8/docs/technotes/tools/unix/javadoc.html#JSSOR647